packages/calcite-components/src/components/filter/filter.scss

```diff
@@ -1,3 +1,17 @@
+/**
+ * @prop --calcite-filter-input-text-color: defines the text color of the input sub-component.
+ * @prop --calcite-filter-input-background-color: defines the background color of the input sub-component.
+ * @prop --calcite-filter-input-border-color: defines the border color of the input sub-component.
+ * @prop --calcite-filter-input-corner-radius: defines the corner radius of the input sub-component.
+ * @prop --calcite-filter-input-placeholder-text-color: defines the placeholder text color of the input sub-component.
+ * @prop --calcite-filter-input-icon-color: defines the icon color of the input sub-component.
+ * @prop --calcite-filter-input-button-border-color: defines the border color of the button in the input sub-component.
+ * @prop --calcite-filter-input-button-background-color: defines the background color of the button in the input sub-component.
+ * @prop --calcite-filter-input-button-background-color-hover: defines the background color of the button when hovered in the input sub-component.
+ * @prop --calcite-filter-input-button-icon-color: defines the icon color of the button in the input sub-component.
+ * @prop --calcite-filter-input-button-icon-color-hover: defines the icon color of the button when hovered in the input sub-component.
+ * @prop --calcite-filter-input-button-icon-color-active: defines the icon color of the button when active in the input sub-component.
```

Comment:
Should we have these vars in alphabetical order? Seems like it would help if they were ordered. Applies to other places as well.